public/js/index.js

```diff
@@ -131,22 +132,46 @@ Index.toggleMode = function () {
     IndexTable.dataTable.draw();
 };
 
-Index.toggleCarousel = function () {
-    localStorage.carouselOpen = (localStorage.carouselOpen === "1") ? "0" : "1";
+Index.toggleCarousel = function (e, firstrun = false) {
```

Comment:
It'd be better to make the parameter more generic than "firstrun" -- Adding documentation to the function would also be useful.

```suggestion
/**
* (documentation)
*/
Index.toggleCarousel = function (e, updateLocalStorage = true) {
if (updateLocalStorage) [...]
```